src/modules/claim/entities/roleClaim.entity.ts

```diff
@@ -1,18 +1,18 @@
 import { Field, ObjectType } from '@nestjs/graphql';
 import { Column, CreateDateColumn, Entity, PrimaryColumn } from 'typeorm';
-import { IRoleClaim, RegistrationTypes } from '../claim.types';
 import { IsArray } from 'class-validator';
 import { Transform } from 'class-transformer';
 import { JWT } from '@ew-did-registry/jwt';
 import { Keys } from '@ew-did-registry/keys';
+import { IRoleClaim, RegistrationTypes } from '../claim.types';
 
 @ObjectType()
 @Entity({ name: 'role_claim' })
 export class RoleClaim implements IRoleClaim {
   static create(data: Partial<RoleClaim>): RoleClaim {
     const entity = new RoleClaim();
     const jwt = new JWT(new Keys());
-    data.subject = jwt.decode(data.token).sub as string;
+    data.subject = (jwt.decode(data.token) as { sub: string }).sub;
```

Comment:
I think this change is fine, but just curious why it is useful. Is it perhaps easier to read in your view? Or is it safer somehow?

Reply:
Basically I updated did-registry and it new version it appear that `jwt.decode` return `unknown`, so Typescript start to mark it as error.

Reply:
Ok, interesting. Fair enough 👍